monai/transforms/utility/dictionary.py

```diff
@@ -52,6 +53,7 @@
     "AsChannelLastd",
     "AddChanneld",
     "RepeatChanneld",
+    "DeleteChanneld",
```

Comment:
The `D` and `Dict` aliases need to go in here as well

Reply:
Oops thought the mapping DeleteChannelD = DeleteChannelDict = DeleteChanneld did that already. Very tricksy, fixed it now

Reply:
Yeah it's not obvious. Thanks!